mimic/rest/maas_api.py

```diff
@@ -118,7 +118,7 @@ def create_entity(params):
     """
     params = collections.defaultdict(lambda: '', params)
     newentity = {}
-    newentity['label'] = params[u'label'].encode("utf-8")
+    newentity['label'] = params['label']
```

Comment:
I was afraid this PR was going to spiral into me having to figure out ReStructuredText :scream: Do we have a doc generator that uses it, or some other way for me to know when I'm getting the syntax correct?

Re: Python string types, let's discuss this more in person at some point. I've done a good amount of Python in school and in previous jobs, but know comparatively little about the internals. I knew there was a difference between `'string'` and `u'string'`, but had to go to Stackoverflow to know that `u'string'` is a Unicode string and `'string'` is a byte string.

The thing that puzzles me is how indexing a dict with `'label'` or `u'label'` gives the same result. If they are two entirely different types of thing, shouldn't they key differently? The only reason I can think of is that it's a terrible UI for a programming language, but the way it is isn't consistent with my mental model of the internals.